src/types.rs

```diff
@@ -10,17 +10,16 @@ pub type Place = Vec<usize>;
 
 /// A symbol for an unspecified term. Only carries meaning alongside a [`Signature`].
 ///
-/// To construct an [`Variable`], use [`Signature::new_var`]
+/// To construct an `Variable`, use [`Signature::new_var`]
 ///
 /// [`Signature`]: struct.Signature.html
-/// [`Variable`]: struct.Variable.html
 /// [`Signature::new_var`]: struct.Signature.html#method.new_var
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub struct Variable {
     pub(crate) id: usize,
 }
 impl Variable {
-    /// A function to return a [`Variable`]'s name
+    /// A function to return a `Variable`'s name
```

Comment:
`A function to` isn't necessary. ``Return a `Variable`'s human-readable name.`` is sufficient.

Go ahead and correct that throughout.